def ErrorCatched(errorsCatched):
  if errorsCatched != []:
    for error in errorsCatched:
      print("El error que sucedio en la ejecucion fue el siguiente: " + error)